src/Symfony/Component/Lock/Tests/Store/BlockingStoreTestTrait.php

```diff
@@ -30,6 +30,9 @@ trait BlockingStoreTestTrait
      *
      * This test is time sensible: the $clockDelay could be adjust.
      *
+     * It also fails when run with the global ./phpunit test suite.
+     * @group transient
```

Comment:
/cc @jderusse FYI

Reply:
last time we had random failure, Cache component emptied the Memcached server wich removes locks and failed the test. Are you sure we don't have conflict with other component here?